### PR TITLE
Specify seperate icon for 'pending' state vs 'running' state

### PR DESCRIPTION
**Pull request by sg00dwin**
And add browse page screenshots for Ashesh

### CODE REVIEW

**Comment by sg00dwin** [comment]
@liggitt pr for review

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -388,6 +388,11 @@ select:invalid {
   }
 }
 
+.spinner.spinner-xs { // set height & width to 12px so that FF displays correctly
```
Comment (liggitt) [inline assets/app/styles/_core.less]: can you open an issue against patternfly with this fix, add a comment with a link to the issue here, and add a TODO to the "Update patternfly" issue so we can track removing our workaround?

**Comment by liggitt** [comment]
LGTM, @smarterclayton merge

**Comment by smarterclayton** [comment]
[merge]

**Comment by sg00dwin** [comment]
@liggitt updated

**Comment by liggitt** [comment]
[merge]

**Pull request merged**